apps/zui/electron-builder-config.js

```diff
@@ -0,0 +1,73 @@
+const { execSync } = require('child_process');
+const zuiPackage = require('./package.json')
+
+const config = {
+  appId: "io.brimdata.zui",
+  asar: true,
+  asarUnpack: ["zdeps", "LICENSE.txt", "acknowledgments.txt", "**/*.node"],
+  directories: {output: "../../dist/apps/zui"},
+  protocols: [{name: "zui", "schemes": ["zui"]}],
+  win: {target: ["nsis"]},
+  linux: {target: ["deb", "rpm"]},
+  rpm: {depends: ["openssl"]},
+  deb: {depends: ["openssl"]},
+  nsis: {oneClick: false, perMachine: false},
+  forceCodeSigning: true,
+  afterSign: "electron-builder-notarize",
+  publish: {
+    provider: "github"
+  },
+  files: [
+    "dist/**",
+    "out/**",
+    "build/**",
+    "zdeps/**",
+    "LICENSE.txt",
+    "acknowledgments.txt",
+    "package.json"
+  ],
+}
+
+// Code below for code signing with SSL.com cert in electron-builder via GitHub
+// Actions taken from:
+// https://github.com/electron-userland/electron-builder/issues/6158#issuecomment-1994110062
+if (process.env.CODE_SIGN_SCRIPT_PATH) {
+  const version = zuiPackage.version;
```

Comment:
The code I borrowed from the GitHub issue was doing this via:

```
const version = execSync('node -p "require(\'./package.json\').version"').toString().trim();
```

I don't know why he didn't opt to just `require` the `package.json` and access the value directly like I'm doing there. But my approach feels cleaner, so I'm going with it. But I figured I'd point it out in case I'm missing something subtle.